src/Tribe/Main.php

```diff
@@ -2988,15 +2988,18 @@ public static function activate() {
 				return;
 			}
 
-			self::flushRewriteRules();
-
 			if ( ! class_exists( 'Tribe__Events__Editor__Compatibility' ) ) {
 				require_once $plugin_path . '/src/Tribe/Editor/Compatibility.php';
 			}
 
 			$editor_compatibility = new Tribe__Events__Editor__Compatibility();
 			$editor_compatibility->deactivate_gutenberg_extension_plugin();
 
+			if ( ! is_network_admin()  ) {
+				// We set with a string to avoid having to include a file here.
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: eb72efbc17be3ede5cf2c9302bbcf508</sub>